.github/workflows/master_doto.yml

```diff
@@ -22,6 +22,7 @@ jobs:
 
     - name: npm install, build, and test
       run: |
+        cd doto-frontend
```

Comment:
@res550 this looks good, however i'm not sure if the deployment will work as I think you will need to cd into doto-frontend after each command as it starts in the root directory i believe

Reply:
Your right the build is currently stalling out we will fix in #28